portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -4547,6 +4547,109 @@ definition {
 			uploadFileName = "Document -_.jpg");
 	}
 
+	@description = "This is a use case for LPS-125462."
+	@priority = "5"
+	test SubmitFormWithUploadFieldAsNewUser {
+		property portal.acceptance = "false";
+		property portal.upstream = "true";
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName();
+
+		Form.editDescription();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Upload");
+
+		var uploadFieldName = FormFields.getFieldName();
+
+		Form.editFieldLabelMetal(fieldValue = "Upload Field");
+
+		Form.closeSidebar();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishCopyURLLink();
+
+		var publishedFormPublicURL = Form.getPublishedFormPublicURL(
+			publishedFormPublicURL = "${publishedFormPublicURL}"
+		);
+
+		Navigator.openURL();
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Users and Organizations");
+
+		User.addCP(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Users and Organizations");
+
+		User.editPasswordCP(
+			userEmailAddress = "userea@liferay.com",
+			userScreenName = "usersn");
```

Comment:
@carolmariaabb, can we edit the created user using another macro to avoid navigate to the user's list again? Is there any available macro to optimize it?

Reply:
Hi @natocesarrego, I could use `Navigator.gotoBack();` (see example [here](https://github.com/carolmariaabb/liferay-portal/blob/8cb974e7d08b8eb1c93ca9b7ad8c164c4502a803/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase#L815)) instead of `ApplicationsMenu.gotoPortlet(..)`, what do you think? 
Unfortunately, I only found this macro to edit the created user and it needs to go back to the user list.

Reply:
No problem, @carolmariaabb. My goal was just understand that we've in our hands. Let's keep it as it's now. 🙂 Thanks.